src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -305,10 +335,11 @@ public void updateForms() {
         .collect(toList()));
   }
 
-  private void chooseLocation(FileChooser fileChooser, JTextField locationField) {
+  private void chooseLocation(FileChooser fileChooser, JTextField locationField, String preferenceKey) {
```

Comment:
I don't quite understand what this is for. Is it maybe for when each form will be able to have its own configuration? Looks like it's not used now.